plotly/grid_objs/grid_objs.py

```diff
@@ -169,12 +160,31 @@ def __init__(self, columns_or_json):
                             "Each column name of your dictionary must have "
                             "'data', 'order' and 'uid' as keys."
                         )
+            # order columns in a list before putting inside the grid
+            ordered_columns = []
+            for order in range(len(columns_or_json['cols'])):
```

Comment:
Ha. So close! Don't depend on order being continuous. The orders might be `2, 5, 6, 19` (AFAIK). We don't depend on that elsewhere. For example, in the front end of the Plotly app, we have a workaround for this:

https://github.com/plotly/streambed/blob/master/shelly/filewell/static/filewell/src/actions/WorkspaceActions.js#L1575-L1579

Can we do something like that here (no opinion on implementation)?